packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -180,7 +174,6 @@ private function doTestFileMatchesExpectedContent(
                 throw $expectationFailedException;
             }
 
-            StaticFixtureUpdater::updateFixtureContent($originalFileInfo, $changedContent, $fixtureFileInfo);
```

Comment:
Instead, this class should be ported here too.

Reply:
ok, ported https://github.com/rectorphp/rector-src/pull/2864/commits/08c08a12974a862207a926e02b9b1f5ef5cdf6ac https://github.com/rectorphp/rector-src/pull/2864/commits/5df08e87635697b52ec1bc7da29eaddf1bfaf955